src/app/games/squarz/SquarzTutorial.ts

```diff
@@ -0,0 +1,116 @@
+const initialState: SquarzState = SquarzRules.get().getInitialState(defaultConfig);
+
+const _: PlayerOrNone = PlayerOrNone.NONE;
+const O: PlayerOrNone = PlayerOrNone.ZERO;
+const X: PlayerOrNone = PlayerOrNone.ONE;
+
+export class SquarzTutorial extends Tutorial {
+
+    public tutorial: TutorialStep[] = [
+
+        TutorialStep.informational(
+            $localize`Squarz`,
+            $localize`Squarz is board control game. Here is the initial state. The goal is to have the most pieces at the end of the game.`,
+            initialState,
+        ),
+
+        TutorialStep.fromPredicate(
+            $localize`Duplication`,
+            $localize`One of the two kinds of move you can do is the duplication. When you do one, you create a new piece. To do this, select one of your pieces, and click on one of its neighboring space.<br/><br/>You're playing Dark, make a duplication.`,
+            initialState,
+            SquarzMove.from(new Coord(0, 0), new Coord(1, 1)).get(),
+            (move: SquarzMove, _state: SquarzState) => {
+                if (move.isDuplication()) {
+                    return MGPValidation.SUCCESS;
+                } else {
+                    return MGPValidation.failure($localize`This was a jump, try to do a duplication.`);
+                }
+            },
+            TutorialStepMessage.CONGRATULATIONS(),
+        ),
+
+        TutorialStep.fromPredicate(
+            $localize`Jumps`,
+            $localize`The second type of move you can do is the jump. When you do one, your piece leaves its original space and jump two spaces further. To do this, select one of your pieces, and click on its landing space two spaces further.<br/><br/>You're playing Light, make a jump.`,
+            new SquarzState([
+                [O, _, _, _, _, _, _, X],
+                [_, O, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _],
+                [X, _, _, _, _, _, _, O],
+            ], 1),
+            SquarzMove.from(new Coord(7, 0), new Coord(5, 2)).get(),
+            (move: SquarzMove, _state: SquarzState) => {
+                if (move.isDuplication()) {
+                    return MGPValidation.failure($localize`This was a duplication, try a jump now.`);
+                } else {
+                    return MGPValidation.SUCCESS;
+                }
+            },
+            TutorialStepMessage.CONGRATULATIONS(),
+        ).withPreviousMove(SquarzMove.from(new Coord(0, 0), new Coord(1, 1)).get()),
+
+        TutorialStep.fromPredicate(
+            $localize`Captures`,
+            $localize`When one of your pieces lands on a square that neighbors an opponent's pieces, those pieces become yours.<br/><br/>You're playing Dark, do such a move!`,
```

Comment:
an opponent, the opponent non ?

Reply:
Euh, centralise les commentaires haha, xlf ou ts mais pas les deux en compets :D